ConnectorAutocadCivil/ConnectorAutocadCivil/Storage/SpeckleStreamManager.cs

```diff
@@ -60,17 +62,18 @@ public static List<StreamState> ReadState(Document doc)
         return streams;
       }
 
-      var record = tr.GetObject(id, OpenMode.ForRead) as Xrecord;
-      var value = GetXrecordData(record);
-
-      try
+      if (tr.GetObject(id, OpenMode.ForRead) is Xrecord record)
       {
-        //Try to decode here because there is old data
-        value = Base64Decode(value);
+        string value = GetXrecordData(record);
+        if (!string.IsNullOrEmpty(value))
+        {
+          //Try to decode here because there is old data
+          if (Base64Decode(value, out value))
```

Comment:
Should it be valid to fail the base64 decode? I appreciate we swallowed it before, but I imagine if we got this far we expected it to work? Should we at least log something?

Reply:
it may be valid to fail in the case of old files with streams saved in an incorrect format (has happened in alpha versions of the connector) in which case we'd want to override the saved streams. will add logging here however